CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+## x.x.x (master)
+
+* Add `!version` command to print bot version from package.json
+* Add CHANGELOG
+
+## 0.1.0 (02-09-2017)
```

Comment:
This is kinda unnecessary but I left it in for an example of how the formatting could look.